src/brogue/MainMenu.c

```diff
@@ -842,7 +842,9 @@ void mainBrogueJunction() {
                             rogue.playbackPaused = false;
                             pausePlayback();
                         }
-
+						if(rogue.gameHasEnded)
+                            break;
```

Comment:
Looks like there are some indentation issues here. Also, what is the purpose of this code?

Reply:
The whole game is played with pausePlayback earlier on the stack. When the game ends it calls nextBrogueEvent from later in the function and requires an extra keypress to exit the game, so I cut it off early. I should probably leave a comment on that.

Reply:
Yes, a comment explaining that when switching to playback, we are within the mainInputLoop called by pausePlayback, so we need to break here before we try to read another event. Also do you think this should be guarded by the playback switch ifdefs, for clarity?

Reply:
Yes, I will do that.